src/AutoRest.CSharp/LowLevel/Output/LowLevelOutputLibraryFactory.cs

```diff
@@ -164,6 +164,11 @@ public static void SetRequestsToClients(IEnumerable<ClientInfo> clientInfos)
                     foreach (var request in operation.Requests)
                     {
                         SetRequestToClient(clientInfo, request, operation);
+
+                        if (operation.Requests.All(r => operation?.RequestMediaTypes?.Values.Contains(r) == true))
```

Comment:
This condition doesn't depend on `request` variable, why is it inside the loop?

Reply:
Because I want to stop processing following requests if this condition is satisfied.

Reply:
But its result is the same every time this expression is executed, so it can be checked before foreach.

Reply:
In fact, we can assume that if `RequestMediaTypes` is not `null`, all requests are added to this dictionary: https://github.com/Azure/autorest/blob/61aff363307502ff422f51e74b3e2edeaa2ff0b3/packages/extensions/modelerfour/src/modeler/modelerfour.ts#L2314